hack/e2e/README.md

```diff
@@ -0,0 +1,50 @@
+# Eventing End-to-End Tests
+Tests the end-to-end flow for eventing-manager.
+
+## Overview
+
+This test covers the end-to-end flow for Eventing. It is divided in three parts:
+1. `setup` - prepares an Eventing CR and verifies that all the required resources are being provisioned by the eventing-manager.
+2. `eventing` - prepares Subscription CR(s) and tests the end-to-end delivery of events for different event types.
+3. `cleanup` - removes the test resources and namespaces from the cluster.
+
```

Comment:
The following content does not reflect these three phases? What exactly do they refer to?

Then again, the command list at the bottom reflects the phases, but without any explanation.

Reply:
I have updated the section.

Reply:
Thank you!